### ISSUE TITLE
Add messages

### ISSUE THREAD

**Issue by mimi89999** [shymmq/librus-client-kotlin#23]

**Comment by RoboBlog** [comment]
https://github.com/RoboBlog/LibrusParser
This parser can be helpful.

**Comment by xdk78** [comment]
yes, thanks but we prefer that code is written in java/kotlin instead python

**Comment by shymmq** [comment]
Combining REST with parser will get very messy very quickly IMO

**Comment by takidelfin** [comment]
Emm, and with larger amount of requests it can slow down... maybe.
